jamovi/surv.a.yaml

```diff
@@ -10,23 +10,24 @@ options:
     - name: data
       type: Data
 
+    - name: elapsed
+      title: Time
+      type: Variable
+      suggested: [ continuous ]
+      permitted: [ numeric ]
+      
+
     - name: event
-      title: Event
+      title: Event indicator (1/0)
```

Comment:
Let's keep this as `Event` ... as we still allow the user to provide nominal variables

Reply:
I updated as suggested.